src/main/java/com/lacunasoftware/pkiexpress/CertificateExplorer.java

```diff
@@ -118,6 +130,10 @@ public CertificateExplorerResult open() throws IOException {
 			this.versionManager.requireVersion(new Version("1.22"));
 		}
 
+		if(trustServiceSession != null){
+			args.add("--trust-service-session");
```

Comment:
I mentioned in the comment above, that is field trustServiceSession is not being passed to `args` to be used in the command